web/src/api/alert.ts

```diff
@@ -0,0 +1,39 @@
+export interface AlertsResult {
+  data: AlertsResultData;
+  status: string;
+}
+
+export interface AlertsResultData {
+  groups: AlertsGroups[];
+}
+
+export interface AlertsGroups {
+  name?: string;
+  file?: string;
+  interval?: number;
+  limit?: number;
+  rules: AlertsRule[];
+}
+
+export interface AlertsRule {
```

Comment:
this could be replaced by https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/src/api/common-types.ts#L167

Reply:
Done, thanks!